### DISCUSSION TITLE
how can I fix it?

### DISCUSSION THREAD

**Discussion by peteryang1756** [upptime/upptime#600] [Q&A] [answered]
Hi, I set to check the website every 5 minutes but my actions still don't run every 5 minutes, how can I fix it?
[https://github.com/sxadxsx/upptime/blob/master/.upptimerc.yml#L62-L69](https://github.com/sxadxsx/upptime/blob/master/.upptimerc.yml#L62-L69)

**Comment by AnandChowdhary** [comment, marked as answer]
Please see https://upptime.js.org/blog/2021/01/22/github-actions-schedule-not-working

**Reply by peteryang1756** [reply]
Hi AnandChowdhary, how to trigger github action using IFTTT

**Reply by AnandChowdhary** [reply]
You can use the IFTTT webhooks service to make an outgoing request, using GitHub's API to dispatch a workflow_dispatch function, for example. :)
